sdk/python/feast/registry.py

```diff
@@ -117,7 +118,7 @@ def get_entity(self, name: str, project: str) -> Entity:
 
     def apply_feature_table(self, feature_table: FeatureTable, project: str):
         """
-        Registers a single feature table with Feast
+        Registers a single feature table or feature view with Feast
```

Comment:
Is this comment change a mistake?

Reply:
oops yes, originally i tried to do one apply method for both but then changed my mind

Reply:
For a moment there I thought you were pulling some series magic shoehorning feature views into feature tables :)